frontend/src/components/Pages/FrontPage/MissionOverview/OngoingMissionCard.tsx

```diff
@@ -29,9 +30,9 @@ const StyledTitle = styled(Card)`
 `
 const HorizontalContent = styled.div`
     display: grid;
-    grid-template-columns: auto auto auto;
+    grid-template-columns: auto auto auto auto;
```

Comment:
```suggestion
    display: flex;
```

Might as well use flex if all are auto

Reply:
If I remove grid as suggested, the items will appear on a vertical line and not a horizontal one. How can the items be placed on a horizontal line without grid?

Reply:
flex-direction: column;